Sources/Helper/Theme.swift

```diff
@@ -101,8 +101,12 @@ public struct Theme {
 
     /// Representation of all customizable visual style property for the `Hyperwallet`'s icon.
     public struct Icon {
-        /// The icon size
+        /// The icon font size
         public static let size = 20
+        /// The icon image height
+        public static let height = 40
```

Comment:
Suggestion: The image is always a square, so you could replace these two variables (`height` and  `width`) to `frameDimention` or `frameSize` = 40.

Reply:
Yes, good idea